web/src/router.js

```diff
@@ -40,6 +39,10 @@ const PATHS = {
   root: "/",
   login: "/login",
   overview: "/overview",
+  installation: "/installation",
+  installationProgress: "/installation/progress",
+  installationFinished: "/installation/finished",
+  logs: "/api/manager/logs.tar.gz",
```

Comment:
@imobachgs 

I've added this path over here, but it smell a bit. If you have a better idea please let me know.